src/main/java/org/example/MainMenu.java

```diff
@@ -0,0 +1,17 @@
+package org.example;
+
+import static org.example.Manager.performOption;
+import static org.example.Menu.showMenu;
+
+public class MainMenu {
+
+    public static String FILE_NAME = "C:\\Java_projekty\\Projeckt_first\\users.txt";
+
+    public static void main(String[] args) {
+
+        for (; ; ) {
```

Comment:
while :) a jak już się uprzesz na takiego for to sformatuj kodzik :)

Reply:
poprawione